### PR TITLE
updating GOVERNANCE.md to change maintainership definitions

### PR DESCRIPTION
**Pull request by gcolliso**
We are updating our Governance to change the maintainership definitions.

### CODE REVIEW

**Comment by ColinSullivan1** [comment]
This looks good to me, much simpler.  When the time is right let's open it up for comments by NATS maintainers + ecosystem and then setup a vote.

File: GOVERNANCE.md [hunk 1]
```diff
@@ -2,26 +2,45 @@
...
+For formal votes, a specific statement of what is being voted on should be added to the relevant Github issue or PR, and a link to that issue or PR added to the maintainers voting document. Maintainers should indicate their yes/no vote on that issue or PR, and after a suitable period of time, the votes will be tallied and the outcome noted.
 
 ## GitHub Project Administration
-Core-maintainers will be added to the NATS GitHub organization and be made an owner of the GitHub organization.
-Each project will have at least one core-maintainer assigned.  Maintainers may get read and write access to a repository with the approval of the appropriate project core-maintainers.
+Maintainers will be added to the NATS GitHub organization.
 
 ## Approving and Merging PRs
-All PRs must receive approval from at least one core-maintainer before merging. PRs can only be merged by core-maintainers.
+All PRs must receive approval from at least two maintainers prior to merging.
```
Comment (derekcollison) [inline GOVERNANCE.md:37]: This would be cumbersome IMO.

**Comment by mmmries** [comment]
:shipit: sounds good to me. Glad we are sticking with 1 maintainer. Smaller projects like the nats.ex adapter don't have a lot of maintainers

**Comment by variadico** [comment]
LGTM

**Comment by aricart** [comment]
LGTM

**Comment by derekcollison** [comment]
LGTM

**Comment by gcolliso** [comment]
This PR to update our Governance Process and Maintainership Definitions has been approved by the NATS Maintainers. Thank you to all who participated.

**Pull request merged**